sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -128,12 +135,19 @@ class AdminOperations {
     }
     
     
-    static func loadSignUpRequests(completion: (([User]) -> Void), failure: (String) -> Void){
+    static func loadSignUpRequests(filter filter: [String: AnyObject]? = nil, completion: (([User]) -> Void), failure: (String) -> Void){
         let manager = BaseOperation.manager()
-        let params = [
+        var params: [String: AnyObject] = [
             NetworkingConstants.kSortAttr: CheckinConstants.kTimeCreated,
             NetworkingConstants.kSortOrder: NetworkingConstants.kDescending
         ]
+
+        if let filter = filter {
+            for (key, value) in filter {
+                params[key] = value
+            }
+        }
+
```

Comment:
should probably make a helper function that adds key value pairs to the params array so we don't have to do it again everytime